### ISSUE TITLE
Import doesn't assign mapped parent

### ISSUE THREAD

**Issue by mauricioarangoosorio** [craftcms/feed-me#1154]
Hi,
I'm testing importing some entries. All of them have to be assigned the same parent. I noticed two failures described below:

1. When I set the parent manually in the mapping page (Feed Element: use default value -  Default Value: The parent entry), the feed runs, the new entries are imported, but they are not assigned the parent I indicated. I don't see any error messages.
2. If I create a column for the parent in the google spread sheet I'm using, and do the mapping to it to assign it to the entry parent, and run the feed the parent is not assigned. I don't see an error message consistently, only once I saw this:
"There was a problem getting the parent element. - Structures.php: 315"

If useful, this is my feed:
```
{
  "range": "textiles!A1:P1000",
  "majorDimension": "ROWS",
  "values": [
    [
      "Name",
      "Image",
      "Parent"
    ],
    [
      "Sample Textitle 4",
      "https://drive.google.com/uc?export=download&id=1OMPH6bZlDuFWpLREb1vVXU2G-Vd5G1nw",
      "All Textiles"
    ],
    [
      "Sample Textile 5",
      "https://drive.google.com/uc?export=download&id=1VElkmzsjsgJOX5SagC3oOb0OwjI1UYug",
      "All Textiles"
    ],
    [
      "Sample Textile 6",
      "https://drive.google.com/uc?export=download&id=1n_M5baz4ZySTvgt0-PGNTkl0QNIcMsM9",
      "All Textiles"
    ]
  ]
}
```

Any advice would be greatly appreciated. Thank you!

System Settings:
PHP version 8.1.3
Linux 5.10.104-linuxkit
MySQL 8.0.26

Craft : Craft Pro 4.0.4
Feed Me: 5.0.4
Freeform: 4.0.0-beta.11
Mailgun: 3.0.0
Ray: 2.0
Redactor: 3.0.0

**Comment by i-just** [comment]
Hi, thanks for reaching out. 

Re 1 - this is a bug, and I’ve created a PR to fix this.

Re 2 - I can’t replicate that issue. To clarify, the parent entry already exists in your database; you’re doing a match on the title, and you’ve selected `title` from the “Data provided for this entry is” dropdown?

**Comment by angrybrad** [comment]
Resolved in https://github.com/craftcms/feed-me/pull/1229

**Comment by RikLamers** [comment]
Hi @i-just ,

I came across this issue since there was a mention of `There was a problem getting the parent element. - Structures.php: 315` in @mauricioarangoosorio description.

I ran into this issue as well. We are working with a sports API to retrieve soccer data. The data we retrieve is match data, when is the match, what teams are competing, where will it be, etc. To get things ordered we use a parent-child construction where the parent is the soccer season and the children are the matches of that season. This parent has to be created by FeedMe when it doesn't exist.

Example:
2022/2023
   competitor 1 - competitor 2
   competitor 1 - competitor 2
   competitor 1 - competitor 2

Everything works fine up until we added the matches for women as well. This is a different competition and within Craft, we made a new section for this to keep things organized. When applying the same settings to the women FeedMe, we got the error mentioned above (`There was a problem getting the parent element. - Structures.php: 315`). After searching for the issue we found out that the slugs of the seasons (2022/2023) were the same. Since we can't have duplicate slugs, the women's feed could be populated correctly.

The solution we applied is to make the season title (2022/2023) unique. But since we want to import it based on the `title` we now have a title with a unique thing in it. It would be nice to have the option to set the `slug` as well as the `title` for example. This wouldn't make sense if you are importing via `slug`, but for other options to import by it would make sense to keep the `title` in this case clean, but have a unique `slug` and prevent the error from occurring and searching for the issue.

To sum up:
We ran into the `There was a problem getting the parent element. - Structures.php: 315` error. Which was caused by a duplicate `slug`. It would be nice to be able to control the `slug` when you don't import by `slug`.

**Comment by i-just** [comment]
Thanks @RikLamers! I can confirm that I can now trigger this error.

**Comment by brandonkelly** [comment]
Feed Me 4.6.0 (Craft 3) and 5.1.0 (Craft 4) have been released with the fix for this.
